.github/workflows/release.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build and push
+
+on:
+  push:
```

Comment:
I guess once the stable branch is created we'll want to add it here, right?
```
branches:
  - stable
```

Reply:
Unsure, the plan is to create a tag for each release.
and deploy using that tag.
No real usage for the "stable" image.

Reply:
IMO, we can keep it per-every-tag. We will just not tag master.

Reply:
ok, sounds good